### PR TITLE
messaging detail page

### PR DESCRIPTION
**Pull request by nguasch**
new page for messaging details
new button rounded
sticker has the same color for candidates and companies
please, review @PablitoGS

### CODE REVIEW

File: html/messaging-detail.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    <div class="wrapper">
+      <div class="container container-expanded">
+        <div class="inner panel-default messaging-detail">
+          <p class="text-center">
+            <span class="sticker-label">Ayer</span>
+          </p>
+
+          <p>
+            <span class="message-out">Hola soy Maria de TickConsulting ¿Te hace una entrevista mañana Donald?</span>
+            <time>12:34</time>
```
Comment (PablitoGS) [inline html/messaging-detail.html]: We could include the datetime attribute

File: html/messaging-detail.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          </p>
+        </div>
+        <div class="panel-canvas messaging-reply inner">
+          <form method="post" action="#">
+            <fieldset>
+              <legend>Lorem ipsum</legend>
+              <ul class="form--field-block clearfix">
+                <li class="field-group-grow">
+                  <div>
+                    <label for="send-message-out" class="hide">Input + button 100% <span class="alternative">(optional)</span></label>
```
Comment (PablitoGS) [inline html/messaging-detail.html]: copy for dev

File: html/messaging-detail.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                    <input id="send-message-out" type="text">
```
Comment (PablitoGS) [inline html/messaging-detail.html]: placeholder

**Comment by nguasch** [comment]
changes form comments done, @PablitoGS

**Comment by PablitoGS** [comment]
👍

**Pull request merged**